### PR TITLE
Nc news 6

### PR DESCRIPTION
**Pull request by YusufKader**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -1,34 +1,138 @@
...
+            })
+          );
+        });
+      });
+  });
+});
+
+describe("PATCH /api/articles/:article_id", () => {
+  const articleUpdate = {
+    votes: 3,
```
Comment (leekli) [inline __tests__/app.test.js]: Double check the Trello board... it wants the object to have a key of `inc_votes`, not `votes`.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -1,34 +1,138 @@
...
+          body: "I find this existence challenging",
+          created_at: "2020-07-09T20:11:00.000Z",
+          votes: 103,
+        });
+      });
+  });
+
+   test("status:404, article not found", () => {
+     return request(app)
+       .get("/api/articles/3000")
```
Comment (leekli) [inline __tests__/app.test.js]: Did you mean for this to be .patch()?

File: __tests__/app.test.js [hunk 3]
```diff
@@ -1,34 +1,138 @@
...
+       .then(({ body }) => {
+         const { message } = body;
+         expect(message).toEqual("Article not found");
+       });
+   });
+
+    test("status: 400, responds with a invalid input", () => {
+      const ARTICLE_ID = "not a number";
+      return request(app)
+        .get(`/api/articles/${ARTICLE_ID}`)
```
Comment (leekli) [inline __tests__/app.test.js]: Did you mean for this to be .patch()?

File: __tests__/app.test.js [hunk 4]
```diff
@@ -1,34 +1,138 @@
...
 
-
 beforeEach(() => seed(data));
 
-
 afterAll(() => {
-    if (db.end) db.end();
-})
+  if (db.end) db.end();
+});
```
Comment (leekli) [inline __tests__/app.test.js]: Please amend this to:

`afterAll(() => {
  return db.end();
});`

File: __tests__/app.test.js [hunk 5]
```diff
@@ -1,34 +1,138 @@
...
+      const ARTICLE_ID = "not a number";
+      return request(app)
+        .get(`/api/articles/${ARTICLE_ID}`)
+        .expect(400)
+        .then(({ body }) => {
+          const { message } = body;
+          expect(message).toEqual("Invalid data type");
+        });
+    });
+});
```
Comment (leekli) [inline __tests__/app.test.js:162]: Please have a think of the following tests for .patch:
* What if the inc_votes value is 0?
* What if the inc_votes value is not a number (such as a string like "hello")?
* What if the vote object does not include the key `inc_vote` ?

**Review by leekli — CHANGES_REQUESTED** [review]
Hi Yusuf,

Some requests for changes attached, see the individual comments in the code.

**Review by yalicee — APPROVED** [review]

**Pull request merged**